scipy/spatial/tests/test_kdtree.py

```diff
@@ -520,6 +521,10 @@ def test_n_jobs():
     with pytest.raises(TypeError, match="Unexpected keyword argument"):
         T.query(points, 1, workers=1, n_jobs=1)
 
+    monkeypatch.setattr(os, 'cpu_count', lambda: None)
+    with pytest.raises(NotImplementedError, match="Cannot determine the"):
+        T.query(points, 1, workers=-1)
```

Comment:
This function is specifically testing the `n_jobs` argument. This belongs in `test_ckdtree_parallel`.